index.html

```diff
@@ -327,8 +327,9 @@
       </section>
       <section id="subscription-response-details">
         <h4>Subscription Response Details</h4>
-        <p>The hub MUST respond to a subscription request with an HTTP [[!RFC7231]] 202 "Accepted" response to indicate that the request was received and will now be verified (<a href="#hub-verifies-intent">Section 4.3 </a>) and validated (<a href="#subscription-validation">Section 4.2 </a>) by the hub. The hub SHOULD perform the verification and validation of intent as soon as possible.</p>
+        <p>If the hub support WebSub and is able to handle the (un)subscription request, it MUST respond to a subscription request with an HTTP [[!RFC7231]] 202 "Accepted" response to indicate that the request was received and will now be verified (<a href="#hub-verifies-intent">Section 4.3 </a>) and validated (<a href="#subscription-validation">Section 4.2 </a>) by the hub. The hub SHOULD perform the verification and validation of intent as soon as possible.</p>
```

Comment:
I think this would be clearer as follows:

"If the hub URL supports WebSub and is able to handle the subscription or unsubscription request..."

* Saying "if the hub supports" is confusing because I think of the hub as software. What's actually meant here is if the hub software is moved to a new location, then it's the hub URL that is left there.
* Better to be explicit about "subscription and unsubscription". I heard some people get confused about "(un)subscription" on the call yesterday.